web/src/unread_ops.ts

```diff
@@ -954,5 +955,8 @@ export function initialize(): void {
         })
         .on("blur", () => {
             window_focused = false;
+
+            // Save the current message as a draft if it exists.
+            drafts.update_draft();
```

Comment:
This code is correct, but from an abstractions standpoint, I think we should setup an independent handler in `compose_setup.initialize`, so it fits together properly.